models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
+::
+ 
+ setenv cesmtag        <your cesm installation folder>
+ setenv resolution     f09_f09_mg17
+ setenv compset        2000_DATM%GSWP3v1_CLM50%BGC-CROP_SICE_SOCN_MOSART_SGLC_SWAV
+ setenv num_instances  5
+
+  ..
+  ..
+
+ if (${num_instances} == 1) then
+   setenv CASE clm5_f09_pmo_SIF
+ else
+   setenv CASE <your tutorial case name>
+ endif
+
+
+Modify the ``SourceModDir`` to match the directory that you set
+in Step 1:
+
+::
+
+ setenv use_SourceMods TRUE
+ setenv SourceModDir   <your SourceMods directory>
+
+
+Confirm the following variables are set to match your personal
+environment, especially ``cesmroot``, ``caseroot``, ``cime_output_root``,
+``dartroot`` and ``project``. 
+
+::
+
+ setenv cesmdata         /glade/p/cesmdata/cseg/inputdata
+ setenv cesmroot         /glade/work/${USER}/CESM/${cesmtag}
+ setenv caseroot         /glade/work/${USER}/cases/${cesmtag}/${CASE}
+ setenv cime_output_root /glade/scratch/${USER}/${cesmtag}/${CASE}
+ setenv rundir           ${cime_output_root}/run
+ setenv exeroot          ${cime_output_root}/bld
+ setenv archdir          ${cime_output_root}/archive
+ ..
+ ..
+ setenv dartroot         /glade/work/${USER}/git/DART_public
+ setenv baseobsdir       /glade/p/cisl/dares/Observations/land
+ ..
+ ..
+ setenv project      <insert project number>
+ setenv machine      cheyenne
+
+
+
+
+Step 12: Execute the Assimilation Run
+-------------------------------------
+
+Set up the assimilation case by executing ``CLM5_setup_assimilation``
+
+::
+
+ cd DART/models/clm/shell_scripts/cesm2_2/
+ ./CLM5_setup_assimilation
+
+.. Caution::
+
+ Once the setup is complete the script will output steps
+ (1-8) displaying 'Check the case'. These steps are good for general reference,
+ however, **for the tutorial ignore these steps and continue to follow the instructions
+ below.**  
+
+After the case is created, by default, it is set up as a 'free' or 'open-loop'
+run.  This means if the case is submitted as is, it will perform a normal CLM 
+simulation without using any DART assimilation.  In order to enable DART
+do the following:
+
+::
+
+ cd <caseroot>
+ ./CESM_DART_config
+
+.. Caution::
+
+ After the script is executed and DART is enabled 'Check the DART configuration:'
+ will be displayed followed by suggested steps (1-5). As before these steps are good for
+ general reference, however, **for the tutorial ignore these steps and continue to follow
+ the instructions below.** 
+
+Now that DART is enabled, confirm, and if necessary, modify the run-time
+settings to perform daily assimilations for 5 total days. Use the
+scripts ``xmlquery`` to view the current settings (e.g. ``./xmlquery STOP_OPTION``)
+and ``xmlchange`` to change the current settings (e.g. ``./xmlchange STOP_OPTION=nhours``).
+Make sure the run-time settings are as follows:
+
+::
+ 
+ DATA_ASSIMILATION_LND=TRUE
+ DATA_ASSIMILATION_SCRIPT= ~/assimilate.csh
+ STOP_OPTION=nhours
+ STOP_N=24
+ DATA_ASSIMILATION_CYCLES=5
+ RESUBMIT=0
+ CONTINUE_RUN=FALSE
+
```

Comment:
It would be easier for the user if we know where these options are. For instance, `env_run.xml`
I'm one of those users who like to view and explore the files in the case directory